qu1cksc0pe.py

```diff
@@ -98,16 +99,19 @@
     libscan = configparser.ConfigParser()
 
 # Utility functions
-def err_exit(message):
-    print(message)
-    sys.exit(1)
+from Modules.utils import err_exit
 
 MODULE_PREFIX = f"{sc0pe_path}{path_seperator}Modules{path_seperator}"
 def execute_module(target, path=MODULE_PREFIX, invoker=py_binary):
+    if "python" in invoker or ".py" in target:
+        # TODO in the future, raise a ValueError/OSError (and remove the additional code below)
+        # instead of warning with a PendingDeprecationWarning
+        DEV_NOTE = "[DEV NOTE]: when switching to import statements, remember to adjust any downstream imports! (e.g. `from .utils import err_exit` vs `from utils import err_exit`)"
+        warnings.warn("Direct execution of Python files won't be supported much longer." + f" {DEV_NOTE}", PendingDeprecationWarning)

```

Comment:
I added a `PendingDeprecationWarning` to be emitted for Python files which get directly executed. Don't worry though, the average user won't see these warnings. They only exist to make it easier for me to hunt down the last remaining direct executions in the future. To make these visible, you have to use a custom warning filter or launch Python in a special mode